draft-ietf-privacypass-auth-scheme.md

```diff
@@ -548,6 +548,16 @@ redeem tokens is maintained and exercised consistently.
 
 # Security Considerations {#sec-considerations}
 
+This section contains security considerations for the PrivateToken authentication
+scheme described in this document.
+
+## Randomness Requirements
+
+Unless otherwise stated, all random values in the challenge and token MUST be
```

Comment:
Are there places where we state otherwise? Or can we just say "all..."

Reply:
Sort of... when we talk about how to construct the redemption context, we say it can be random, but can also be something else. We are probably fine without the precondition though, so I'll just drop it.